### ISSUE TITLE
Market Maker is configured in package.json commands

### ISSUE THREAD

**Issue by TimDaub** [leapdao/exit-market-maker#19]
## Scope

- See this file: https://github.com/leapdao/exit-market-maker/blob/master/package.json#L11
- Configuration is done by formatting a JSON object such that it gets passed as an environment variable through the command line to the market maker
- serverless recommends to store variables securely in the AWS SSM parameter store: https://serverless.com/framework/docs/providers/aws/guide/variables/

## Deliverables

- Store and read configuration variables from SSM

## Roles
bounty gardener: @TimDaub / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

**Comment by troggy** [comment]
I don't think it worth a hassle as these settings are not sensitive — nothing private. It may only be a matter of convenience: if config is in SSM you can redeploy lambda without the risk of overwriting the current settings
